tests/snapshots/tests.contrib.mariadb.test_mariadb.test_query_proc_snapshot.snap

```diff
@@ -0,0 +1,22 @@
+[[{"name" "mariadb.query"
+   "service" "mariadb"
+   "resource" "sp_sum"
```

Comment:
Interesting that the resource for a procedure ends up getting truncated to this... @brettlangdon any thoughts?

Reply:
this is probably why: https://github.com/DataDog/dd-trace-py/blob/0dc356f0a672f61383766066de068115d2962fb7/ddtrace/contrib/dbapi/__init__.py#L115

because it is `cursor.callproc("proc_name")` we don't have the query, that is stored on the server

I am surprised we only have 1 span here since this snapshot tests has two executes and the callproc ?

Reply:
@Kyle-Verhoog any further thoughts on this?

Reply:
hmm yeah haven't thought about it any further but it's definitely non-blocking for this PR

Reply:
ah for some reason i didn't realize that `sp_sum` was the given name of the stored procedure.. seems totally fine to me :+1:

> I am surprised we only have 1 span here since this snapshot tests has two executes and the callproc ?

lol yeah the test for some reason disables the tracer for the two executes: https://github.com/DataDog/dd-trace-py/pull/2570/files#diff-b5666b23f77bb419456af9d9b0a778a1a787fafbee39303612e5de24e4960500R264